index.html

```diff
@@ -1,72 +1,225 @@
-<!DOCTYPE html>
-<html lang="en">
-<!---
-	A TEMPORARY landing page for the SG-R project.
-	Thrown together in <10 mins via the Gramateria framework
-	-->
-
-<head>
-    <meta name="title" content="The Stargate-Rewritten Project">
```

Comment:
Given that this page is most frequently viewed in discord, it makes sense to keep a lot of this meta.
Notably, title, author, the og flags, a theme colour, and an icon.